wave_vision/include/wave/vision/tracker/tracker.hpp

```diff
@@ -50,7 +60,7 @@ class Tracker {
      * @return tracks corresponding to all detected landmarks in the image, from
      * the start of time to the given image.
      */
-    std::vector<FeatureTrack> getTracks(const size_t &img_num) const;
+    std::vector<FeatureTrack> getTracks(const int img_num) const;
```

Comment:
int?

Reply:
changed. I can't really remember why I changed it. I think there will be some significant changes coming here as we start using this again.